src/sidebar/components/AutocompleteList.tsx

```diff
@@ -36,19 +51,16 @@ const defaultListFormatter = item => item;
  * used by itself.
  *
  * Modeled after the "ARIA 1.1 Combobox with Listbox Popup"
- *
- * @template T
- * @param {AutocompleteListProps<T>} props
  */
-export default function AutocompleteList({
+export default function AutocompleteList<Item>({
```

Comment:
Is it OK to use a named generic here that represents more semantically what this component operates on (which is a list of "items")?

Reply:
Yes, I think this is sensible. `T` is fine if the role of the type is obvious, but in this case I agree that a more verbose name is helpful.